docs/getting-started/development.md

```diff
@@ -0,0 +1,38 @@
+# Development
+
+It's nice to see you here. Every contribution is always more than welcome and very appreciated.
+The current development process for Spectral might seem to be complicated and make you feel overwhelmed.
+
+We prepared a common list of gotchas and things to keep in mind when writing any code.
+
+## Introduction
+
+Although, it might not be very obvious, Spectral is meant to be run in any ES2017 and above JS environment, therefore there are a few rules you need to be aware of.
+One of the main principle is to be very strict on using any Node.js exclusive modules, so if you can help it, avoid using it.
+The same applies to globals specific to particular environment, such as fetch in a browser, etc.
+In general, Spectral should run in a similar manner regardless of the environment it's executed in.
```

Comment:
I think you need new lines between paragraphs in order for markdown to know they are different paragraphs dont you?

Reply:
Yeah, though I just added that whitespace to make writing easier :sweat_smile: 
Will make it easier to read now!